server/routes/allCompletedReports/allCompletedReports.ts

```diff
@@ -43,6 +47,16 @@ export default class AllCompletedReportsRoutes {
   view = async (req: Request, res: Response): Promise<void> => {
     return this.validateRoles(req, res, async () => {
       const uiFilter = fillInDefaults(uiFilterFromRequest(req))
+      if (uiFilter.status === undefined) {
+        /* eslint-disable @typescript-eslint/no-explicit-any */
+        uiFilter.status = [
+          ReportedAdjudicationStatus.AWAITING_REVIEW,
+          ReportedAdjudicationStatus.UNSCHEDULED,
+          ReportedAdjudicationStatus.RETURNED,
+          ReportedAdjudicationStatus.SCHEDULED,
+          ReportedAdjudicationStatus.REJECTED,
+        ] as any
+      }
```

Comment:
I think this default array needs to be in the fillInDefaults function as I suggested on slack